nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/nf-registry.js

```diff
@@ -21,7 +21,9 @@ import NfRegistryService from 'services/nf-registry.service';
 import NfStorage from 'services/nf-storage.service';
 import nfRegistryAnimations from 'nf-registry.animations';
 import NfRegistryApi from 'services/nf-registry.api';
+import { NfRegistryExplorerAbout } from 'components/explorer/dialogs/about/nf-registry-explorer-about';
```

Comment:
OK to remove if the component will not be used in `showAboutDialog` (see my comments about this method).